package.json

```diff
@@ -8,20 +8,26 @@
     "start": "next start",
     "lint:js": "next lint",
     "lint:css": "stylelint '**/*.scss'",
-    "lint": "run-s lint:js lint:css"
+    "lint": "run-s lint:js lint:css",
+    "test": "jest", 
+    "test:watch": "npm run test -- --watchAll"
   },
   "dependencies": {
     "next": "12.3.1",
     "react": "18.2.0",
     "react-dom": "18.2.0"
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "jest-environment-jsdom": "^29.1.2",
     "eslint": "8.24.0",
     "eslint-config-next": "12.3.1",
+    "jest": "^29.1.2",
     "npm-run-all": "^4.1.5",
+    "sass": "^1.55.0",
```

Comment:
💯 Great fix with this! Dependencies should be listed alphabetically automatically....it's possible this got resolved incorrectly.